src/apps/loan-list/materials/stackable-material.tsx

```diff
@@ -70,121 +84,126 @@ const StackableMaterial: FC<StackableMaterialProps & MaterialProps> = ({
+          additionalMaterials > 0 ? "list-reservation--stacked" : ""
+        }`}
+      >
+        <div className="list-reservation__material">
           <div>
-            <div className="status-label status-label--outline">{specific}</div>
-          </div>
-          <div className="list-reservation__about">
-            <h3 className="text-header-h4">{mainText}</h3>
-            <p className="text-small-caption color-secondary-gray">
-              {creators &&
-                getAuthorNames(
-                  creators,
-                  t("loanListMaterialByAuthorText"),
-                  t("loanListMaterialAndAuthorText")
-                )}
-              {year?.year && <> ({year.year})</>}
-            </p>
+            <Cover
+              pid={pid as Pid}
+              size="small"
+              animate={false}
+              description={abstract && abstract[0]}
+            />
           </div>
-          {additionalMaterials > 0 && (
-            <>
-              <div
-                className="list-reservation__hidden-explanation"
-                id="materials-modal-desktop-text"
-              >
-                {t("loanListMaterialsModalDesktopText")}
+          <div className="list-reservation__information">
+            <div>
+              <div className="status-label status-label--outline">
+                {specific}
               </div>
-              <button
-                type="button"
-                onClick={(e) => {
-                  openDueDateModal(e);
-                }}
-                aria-describedby="materials-modal-desktop-text"
-                id="test-more-materials"
-                className="list-reservation__note-desktop text-small-caption color-secondary-gray"
+            </div>
+            <div className="list-reservation__about">
+              <h3 className="text-header-h4">{mainText}</h3>
+              <p className="text-small-caption color-secondary-gray">
+                {creators &&
+                  getAuthorNames(
+                    creators,
+                    t("loanListMaterialByAuthorText"),
+                    t("loanListMaterialAndAuthorText")
+                  )}
+                {year?.year && <> ({year.year})</>}
+              </p>
+            </div>
+            {additionalMaterials > 0 && (
+              <>
+                <div
+                  className="list-reservation__hidden-explanation"
+                  id="materials-modal-desktop-text"
+                >
+                  {t("loanListMaterialsModalDesktopText")}
+                </div>
+                <button
+                  type="button"
+                  onClick={(e) => openDueDateModalCallBack(e)}
+                  aria-describedby="materials-modal-desktop-text"
+                  id="test-more-materials"
+                  className="list-reservation__note-desktop"
+                >
+                  + {additionalMaterials} {t("loanListMaterialsDesktopText")}
```

Comment:
I imagine I'll fix the strings (there are others) once we settle the concat versus interpolate discussion :) because this really should be interpolated imo